app/services/txt2sql_engine.py

```diff
@@ -324,4 +324,22 @@ def txt2sql_engine(self,user,user_prompt,thread_id):
         else:
             response={"result":explanation}
             result={'success':False,'message':explanation,'data':[response]} 
-        return result
\ No newline at end of file
+        return result
+    
+    def get_table_columns(self, user, table_name):
```

Comment:
```pythonimport json

def get_table_columns(self, user, table_name):``` ``criticality: MAJOR `` 

 > Missing import statement for 'json' module. This will cause a NameError when the json.loads() method is called.